setup.py

```diff
@@ -109,7 +109,7 @@ def can_import(module_name):
 # (and likely will pin specific version numbers), using
 # setup.py's install_requires is preferred for a library
 # (and should try not to be overly narrow with versions).
-REQUIRES = ["numpy"]
+REQUIRES = ["numpy", "typing_extensions"]
```

Comment:
Wouldn't this introduce a new requirement for all users?

Reply:
Reading https://pypi.org/project/typing-extensions/ suggests is is there officially for Python 3.7 onwards. Are you clearer on what the situation is on Python 3.6 where this is "provisional"?

Reply:
Python 3.7 and 3.8 don't have typing_extensions.

Reply:
good point, I forgot that `TYPE_CHECKING` is part of `typing` in Python 3.6+ (only Python 3.5 needs the `typing_extensions` for this)

Reply:
@mdehoon `typing_extensions` was needed to enable `mypy Bio` to run without error. It is `pip` installable on Python 3.7 and Python 3.8 (and presumably Python 3.9) from PyPI.

@peterjc `typing-extensions` is a backport for `typing` features from Python 3.7 to Python 3.6 and 3.5, so all is good.